data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1853 @@
+				accept
+			label end
+			`	"Appreciate your willingness to assist us, I do, Captain. Until we meet again." He thanks you once more and ends the call.`
+
+mission "Resistance: Evacuation 3"
+	name "Pick Up Desert Ranchers"
+	description "Head to <destination> to pick up the <bunks> ranchers by <date> and bring them back to their world, Shifting Sand."
+	deadline 8
+	passengers 47
+	source
+		not government "Heliarch"
+		near "Silver Bell" 2 3
+	destination "Chosen Nexus"
+	to offer
+		random < 50
+		has "Resistance: Evacuation 2: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			branch resistance
+				has "Resistance: Questions: done"
+			`Walking around the spaceport for a few minutes, you're met by a pair of Saryds. "Captain <last>? Worked with our company before, you have. Helped with transporting a group of students, you had." You nod. "Another troubling situation, we have. Help us again, would you?" They point to a Kimek Spire, presumably the very same you boarded last time, parked at the edge of the local docks.`
+			choice
+				`	"Sure, lead the way."`
+					goto talk
+				`	"I'm not interested in that kind of job at the moment, sorry."`
+			`	"Certain of that, are you? Willing to pay very well for your continued service, the captain is. Change your mind, would 748,300 credits?"`
+			choice
+				`	"On second thought, I'd love a transport job right now."`
+				`	"I'm just not interested in working with you anymore, no."`
+					decline
+			label talk
+			`	Once again you're brought to the Spire, but only the Saryds come in with you, unlike the Kimek from last time. You're all escorted through the corridors, and are all searched by the devices, before you're finally brought to Elirom. He greets you, thanking you for coming, "Some more problems, we've had, so in need of a contractor's help once more, the company is.`
+			`	"Contracted us for transport to <planet>, some Arach ranchers did. Attempting to start a longcow business on the deserts of Shifting Sand, they are, and hoping for a loan from a prestigious bank on <planet>, to start off a new expansion to the ranch they were. <bunks> they are, in total, both ranchers and their employees. Only..." He pauses for a moment, as if considering something. "Difficult, the Arachi can be at times. Once discovered they had, that originally from a rival Arach House, the captain of their original transport was, refuse to return with him, they did. Given us until <date> to find a new transport, before they cancel out on us, they have."`
+			`	He says you'll receive 748,300 credits once the ranchers are back on Shifting Sand, and urges you to hurry to <destination> before <date>, so that the Arachi don't cancel the transport. He wishes you good luck, and you're brought back out of the ship. The Saryds that went in with you stay inside, and the Spire departs not long after you've left.`
```

Comment:
```suggestion
			`	He says you'll receive <payment> once the ranchers are back on Shifting Sand, and urges you to hurry to <destination> before <date>, so that the Arachi don't cancel the transport. He wishes you good luck, and you're brought back out of the ship. The Saryds that went in with you stay inside, and the Spire departs not long after you've left.`
```